### PR TITLE
[Remove Stream/Optional usage] LPS-174212 Remove_usages_of_Stream_usages_for_module_roles_test

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

This is for ticket https://issues.liferay.com/browse/LPS-174212.
Thanks for checking!

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:roles

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/roles/roles-test/src/testIntegration/java/com/liferay/roles/service/test/RoleLocalServiceTest.java [hunk 1]
```diff
@@ -271,39 +269,33 @@ public void testGetGroupRolesAndTeamRoles() throws Exception {
 			companyId, null, excludedRoleNames, roleTypes, 0, groupId,
 			QueryUtil.ALL_POS, QueryUtil.ALL_POS);
 
-		List<Role> expectedRoles = _roleLocalService.getRoles(companyId);
+		List<Role> expectedRoles = ListUtil.filter(
+			_roleLocalService.getRoles(companyId),
+			role -> {
+				if (!excludedRoleNames.contains(role.getName()) &&
```
Comment (dantewang) [inline modules/apps/roles/roles-test/src/testIntegration/java/com/liferay/roles/service/test/RoleLocalServiceTest.java:275]: @jeffwu0724 Could you try to flip the conditions to buy spaces? Thanks!

**Comment by dantewang** [comment]
@jeffwu0724 Please see my comment and resend. Thanks!

**Pull request closed without merging**